samples/nrf9160/http_application_update/Kconfig

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "HTTP application update sample"
+
+config DOWNLOAD_HOST
+	string "Download host name"
+
+config DOWNLOAD_FILE
+	string "The file to download"
+
+config FLASH_WRITE_OFFSET
+	hex
+	default 0x80000
```

Comment:
Why not take this information automatically from DT?
- `DT_FLASH_AREA_IMAGE_0_OFFSET`
- `DT_FLASH_AREA_IMAGE_0_SIZE`

That way it can run out-of-box on all boards.

Reply:
Fixed now.